include/benchmark/benchmark.h

```diff
@@ -1304,7 +1304,7 @@ class BENCHMARK_EXPORT Benchmark {
  public:
   const char* GetName() const;
   int ArgsCnt() const;
-  const char* GetArgName(int arg) const;
+  const char* GetArgName(size_t arg) const;
```

Comment:
I'm not sure about this one, we return signed and then take unsigned.
I think things should generally be signed, but even then, this seems unconsistent

Reply:
yeah i didn't feel great about that either.  i'll tweak it.